Automattic/wp-calypso
# Pricing page: show 'Included Products' section to all locale

#### Proposed Changes

* Use hook for included product description map instead of helper function to avoid issue w/ translation.
* Remove locale restriction for `Included Product` section in the bundle lightbox.
**old**
<img width="1101" alt="Screen Shot 2022-12-12 at 3 22 25 PM" src="https://user-images.githubusercontent.com/56598660/206984700-020c0352-9b38-431a-b35b-3888d6eae458.png">

**new**
<img width="1114" alt="Screen Shot 2022-12-12 at 3 21 04 PM" src="https://user-images.githubusercontent.com/56598660/206984588-91518134-8851-4a91-9d64-0ac31857ec16.png">

#### Testing Instructions

1. Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
    * Run `git fetch && git checkout remove/locale-restriction-for-included-products-info`
    * Run `yarn start-jetpack-cloud`

2. Scroll down to the footer and change locale to different one other than 'English'.
<img width="1168" alt="Screen Shot 2022-12-12 at 3 24 45 PM" src="https://user-images.githubusercontent.com/56598660/206985168-747df738-f2b8-49df-ad5a-5a8100c0b5a3.png">

3. Click **More about Complete** and **More about Security** link to display bundle lightbox.
<img width="1228" alt="Screen Shot 2022-12-12 at 3 26 40 PM" src="https://user-images.githubusercontent.com/56598660/206985470-b0267d8a-8839-4b0b-8a18-0617b2288d5b.png">

4. Confirm that the included products section is displayed.

#### Pre-merge Checklist
